charts/generic-data-analytics-extractor/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+name: generic-data-analytics-extractor
+description: Runs a daily cron job which extracts and sends dump of database to the analytical platform
```

Comment:
sends a dump of the database?
sends a database dump?

Reply:
I'll go for sends a database dump